src/logic/operate.js

```diff
@@ -0,0 +1,29 @@
+const Big = require('big.js');
+
+const Operate = (numberOne, numberTwo, operation) => {
+  const firstNum = Big(numberOne);
+  const secondNum = Big(numberTwo);
+  let finalResult;
+
+  switch (operation) {
+    case '+':
+      finalResult = firstNum.plus(secondNum);
+      break;
+    case '-':
+      finalResult = firstNum.minus(secondNum);
+      break;
+    case 'X':
+      finalResult = firstNum.times(secondNum);
+      break;
+    case '/':
+      if (secondNum === '0') {
+        finalResult = '0';
+      }
+      finalResult = firstNum.div(secondNum);
+      break;
```

Comment:
return `INFINITY`, or `NAN` ........it's no Zero ;)

Reply:
Thank you :) I fixed it.